lua/orgmode/colors/markup_highlighter.lua

```diff
@@ -423,8 +442,10 @@ local function apply(namespace, bufnr, line_index)
 end
 
 local function setup()
-  for _, marker in pairs(markers) do
-    vim.cmd(marker.hl_cmd)
+  for delimiter, marker in pairs(markers) do
+    vim.cmd(string.format(marker.hl_cmd, marker.hl_name))
+    vim.cmd(string.format('syn keyword %s_delimiter %s', marker.hl_name, delimiter))
+    vim.cmd(string.format(marker.hl_cmd, marker.hl_name .. '_delimiter'))
```

Comment:
We don't need delimiters highlight for `latex` type since we do not handle those markers as markup and we do not add extmarks for them.

Reply:
Done, added an option in the markers to enable or disable the delimiter feature and do a check for it both in the `hl` group creation and in the `extmarks`.

You may actually want to review that in particular, it feels maybe kinda hacky to me or maybe I'm just stupid. Could be one or the other 🥲.